mattermost-plugin/Makefile

```diff
@@ -277,6 +285,36 @@ ifndef STARTERTEMPLATE_PATH
 endif
 	cd ${STARTERTEMPLATE_PATH} && go run ./build/sync/main.go ./build/sync/plan.yml $(PWD)
 
+## Watch webapp and server changes and redeploy locally using local filesystem (MM_SERVER_PATH)
+.PHONY: live-watch
+live-watch:
```

Comment:
If this target is only planning to affect and act over plugins, I'd make reference to it both in the docline and in the name, maybe changing it to something like `make live-watch-plugin`

Why not adding this target to the root's `Makefile`? This way we could run all the watch targets from the root of the repo

Reply:
Sounds sensible, I was trying to keep that there because was the plugin part, I think we can just make a new target on the root makefile that delegates the work to this.

Reply:
Finally I just exposed that in the Makefile